packages/components/src/components/typography/Link/Link.tsx

```diff
@@ -99,6 +105,8 @@ const Link = ({
             $variant={variant}
             className={className}
             {...textProps}
+            $color={color}
+            $gap={gap}
```

Comment:
I think it doesn't belong here, couldn't you use Row inside of the link?

something like:

```
<Link>
  <Row as="span" gap={spacings.xs}>
    <A/>
    <B/>
  </Row>
</Link>

Reply:
Again it's here because I must remove the gap and add a space to preserve the underline